src/projections/isea.cpp

```diff
@@ -949,8 +948,7 @@ static int isea_hex(struct isea_dgg *g, int tri,
         hex->y = i + sidelength * ((g->quad-1) % 5);
     }
 
-    return 1;
-#endif
+    return;
```

Comment:
Not needed any more.

Reply:
Sorry, I don't follow.  What's not needed any more exactly?  If we put the `#endif` back in, it will not be valid because its matching `#ifdef` has been removed.

Reply:
I believe @rhuijben meant that the explicit "return;" does nothing at the end method and can be safely removed

Reply:
Thanks for clearing that up @rouault.  I have removed the 'return' as suggested.